README.md

```diff
@@ -0,0 +1,143 @@
+# `dss-conduits`
+
+Part of this code was inspired by https://github.com/makerdao/rwa-toolkit/blob/master/src/urns/RwaUrn.sol mainly authored by livnev and https://github.com/dapphub/ds-roles/blob/master/src/roles.sol authored by DappHub.
+Since it should belong to the MakerDAO community the Copyright from our additions has been transferred to Dai Foundation.
+
+## Overview
+Implementation of the allocation system, based on the [technical specification forum post](https://forum.makerdao.com/t/preliminary-technical-specification-of-the-allocation-system/20921
+).
+
+![Untitled](https://github.com/makerdao/dss-allocator/assets/130549691/388f20fa-2d0c-484b-b716-fe4fa742115b)
```

Comment:
Not sure I follow why the dao proxy only connects to the buffer.

Reply:
Also registry is missing here.

Reply:
The lines where supposed to show movements of funds, so the assumption was that the subdao proxy would only transfer or transferFrom the buffer. Anyway I removed that link to avoid confusion as that is not stated explicitly.

Also Added the registry (not sure we must include all the contracts so for now I hope it doesn't add too much info on the eye).